extensionsGallery.json

```diff
@@ -4077,6 +4077,63 @@
 					"statistics": [],
 					"flags": ""
 				},
+				{
+					"extensionId": "81",
+					"extensionName": "sql-migration",
+					"displayName": "Azure SQL Migration",
+					"shortDescription": "This extension lets you migrate your SQL Server instance to Azure SQL.",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "0.1.8",
+							"lastUpdated": "08/18/2021",
```

Comment:
fix the date

Reply:
this is the same extension build as the one released on the insider's on 8/18. should it still be updated to today's or 9/1 date?

Reply:
not a huge deal, i guess it is fine